mods/ra/rules/aircraft.yaml

```diff
@@ -117,9 +117,8 @@ MIG:
 		RepulsionSpeed: 40
 		MaximumPitch: 56
 	AutoTarget:
-		TargetWhenIdle: false
-		TargetWhenDamaged: false
-		EnableStances: false
+		InitialStance: HoldFire
+		InitialStanceAI: HoldFire
```

Comment:
Yes, this is a manual adjustment as we actually want aircraft to have stances enabled. (I'd only need `EnableStances: false` to retain the old [bogus] behavior.)